packages/bcer-api/app/src/manufacturing/manufacturing.service.ts

```diff
@@ -101,4 +101,10 @@ export class ManufacturingService {
     await this.manufacturingRepository.delete({})
     return;
   }
+
+  async assignManufacturingToNewBusiness(currentBusinessId: string, newBusinessId: string){
+    const result = await this.manufacturingRepository
+    .query(`UPDATE manufacturing SET "businessId" = $1 WHERE "businessId" = $2`, [newBusinessId, currentBusinessId])
```

Comment:
Any reason for using a raw query here instead of the `queryBuilder` as done in the location service?

Reply:
(similar question for the other update statements)